onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder_helpers.h

```diff
@@ -181,8 +181,8 @@ Status AddMinMaxOperator(ModelBuilder& model_builder, const NodeUnit& node_unit,
 Status AddReshapeOperator(ModelBuilder& model_builder, const NodeUnit& node_unit,
                           const std::string& input, const std::vector<int32_t>& shape);
 
-bool CanSkipReshape(const ModelBuilder& model_builder, const NodeUnit& node_unit,
-                    size_t input_rank, size_t output_rank);
+// bool CanSkipReshape(const ModelBuilder& model_builder, const NodeUnit& node_unit,
+//                     size_t input_rank, size_t output_rank);
```

Comment:
nit: if it was only used in op_builder_helpers.cc, maybe just make it a (commented out) function with internal linkage in op_builder_helpers.cc? one less place to update.